debug/bright-v7-types.json

```diff
@@ -0,0 +1,1324 @@
+{
+  "version": 7,
+  "name": "Bright",
+  "constants": {
+    "@name": { "type": "field", "value": "{name_en}" },
```

Comment:
Yep, not relying on it, just visually checking what all the types should look like — fire away.